src/Components/homepage.css

```diff
@@ -0,0 +1,134 @@
+@import url('https://fonts.googleapis.com/css2?family=Lato:wght@100&family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;1,100;1,200&display=swap');
+
+body {
+  background-image: url(/src/assets/bgg.jpg);
+  background-size: cover;
+  font-family: 'poppins', sans-serif;
+}
+
+.main-container {
+  display: grid;
+  grid-template-columns: repeat(2, 1fr);
+  grid-gap: 2px;
+}
+
+.header {
+  height: 180px;
+  background-image: url(/src/assets/bgg.jpg);
+  border-radius: 10px;
+}
+
+.header h1 {
+  font-size: 30px;
+  color: rgba(230, 243, 255, 0.459);
+  font-weight: 600;
+  margin-top: 30px;
+  text-align: left;
+  padding: 2%;
+}
+
+.country-container {
+  background-color: #0072ff6e;
+  margin: 2px 2px;
+  height: 150px;
+  border-radius: 10px;
+  border: 1px solid #bad9ff3a;
+}
+
+.flag {
+  width: 20%;
+  height: 20px;
+  margin-right: 5%;
+}
+
+ul {
+  padding: 0%;
+}
+
+.country-name {
+  width: 100%;
+  display: flex;
+  list-style: none;
+  align-items: center;
+}
```

Comment:
Nice and beautiful design so far, however, as stated in the project requirements, you are suppose to follow the design guidelines including: 

- [x] Colors (select one main color for your website).
- [x] Typography: font face, size and weight.
- [x] Layout: composition and space between elements.

There are some differences between yours and that of the actual design which you will need to fix.

1.  The home page is actually suppose to be in two rows with a light and dark color alternating between the cards on each column.  
2. Also, there are no spaces between the cards
3. Kindly check the position and size of the country image/flag, country name, as well as the small arrows on the top right corner of each country card.



### Your design 


![Screenshot from 2023-02-09 22-48-49](https://user-images.githubusercontent.com/65233860/217957266-9d9ec757-eb63-4f28-a53c-38e18b563484.png)


### The actual design 

![Screenshot from 2023-02-07 18-49-55](https://user-images.githubusercontent.com/65233860/217398126-e4f1d1b4-53cf-4f6c-b918-6e8cf6854c94.png)

Reply:
- [ ] This issue is well resolved :heavy_check_mark: